pkg/controller/apimanagerrestore/apimanagerrestore_logic_reconciler.go

```diff
@@ -444,18 +354,8 @@ func (r *APIManagerRestoreLogicReconciler) reconcileRestoreAPIManager() (reconci
 
 	existing := &appsv1alpha1.APIManager{}
 	err = r.ReconcileResource(existing, apimanager, reconcilers.CreateOnlyMutator)
-	if err != nil {
-		return reconcile.Result{}, err
-	}
-
-	stepFinished := true
-	r.cr.Status.APIManagerRestoreStepFinished = &stepFinished
-	err = r.UpdateResourceStatus(r.cr)
-	if err != nil {
-		return reconcile.Result{}, err
-	}
+	return reconcile.Result{}, err
 
-	return reconcile.Result{}, nil
 }
 
 func (r *APIManagerRestoreLogicReconciler) reconcileAPIManagerBackupSharedInSecretCleanup() (reconcile.Result, error) {
```

Comment:
This method is related to what's been commented in reconcileAPIManagerBackupSharedInSecretCleanup. Notice that in the Reconcile() function this is one of the latest steps